Automattic/jetpack
# SSO: remove jQuery dependency for improved performance

## Proposed changes:

This PR brings in 2 changes:

1. It removes the need for jQuery for the different js files added to the log in form and wp-admin for SSO.
2. It switches to using the new utility introduced in #36572.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings > Security and enable SSO
* Go to the Users menu
* Move your mouse hover the SSO "[?]" and "!" icons, and ensure the tooltips appear.
* Go to Users > Add New
* Click on the "Invite user to WordPress.com" checkbox
    * The "Custom message" field should appear.
* Now log out of the site
* On the SSO login screen that appears, ensure that things still work as expected.
